src/main/java/io/jenkins/plugins/artifact_manager_jclouds/s3/S3BlobStoreConfig.java

```diff
@@ -58,6 +58,8 @@
 
 import jenkins.model.Jenkins;
 import org.jenkinsci.Symbol;
+import jenkins.security.FIPS140;
```

Comment:
nit: better to keep order
```suggestion
import jenkins.model.Jenkins;
import jenkins.security.FIPS140;
import org.jenkinsci.Symbol;
```